model/exclusionList/ExclusionListClassService.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+declare(strict_types=1);
+
+namespace oat\taoClientDiagnostic\model\exclusionList;
+
+use core_kernel_classes_Class;
+use core_kernel_classes_Property;
+use core_kernel_classes_Resource;
+use oat\generis\model\OntologyRdfs;
+use oat\tao\model\OntologyClassService;
+
+/**
+ * Class exclusionListClassService
+ *
+ * @package oat\taoClientDiagnostic\model\exclusionList
+ */
+abstract class ExclusionListClassService extends OntologyClassService
```

Comment:
I agree, actually, this structure is a port of a similar feature supplied by another TAO extension, and I merely reused the same naming conventions... Since this PR is adding these files, better to correct the names now. So I will ;)